Src/VimCore/CommandUtil.fs

```diff
@@ -1603,7 +1609,8 @@ type internal CommandUtil
             CommandResult.Completed ModeSwitch.SwitchPreviousMode
 
     /// Invert the current selection
-    member x.InvertSelection (visualSpan: VisualSpan) (streamSelectionSpan: VirtualSnapshotSpan) columnOnlyInBlock =
+    member x.InvertSelection (visualSpan: VisualSpan) columnOnlyInBlock =
+        let streamSelectionSpan = _textView.Selection.StreamSelectionSpan

```

Comment:
This doesn't appear to be used.

Reply:
Not sure which identifier you mean. If you mean `x.InvertSelection`, it is definitely used by `VisualCommand.InvertSelection`. If you mean `streamSelectionSpan`, it is definitely used about five lines later in the local function `changeSelection`. In general you can tell if something is unused with the F# language service because it will always be "grayed out", i.e. shown in a fainter tag color.

Reply:
I was referring to `streamSelectionSpan` and I just missed that it was used later. The F# language service does make this obvious for general editting but it doesn't work during code reviews (that I know of).